Task2.Tests/Task2.Tests.csproj

```diff
@@ -0,0 +1,19 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
```

Comment:
It's not required but,
https://devblogs.microsoft.com/dotnet/announcing-dotnet-7/  You could try .net 7 or 6

Reply:
Okay, if you dont mind I will try it in next task)